src/rules/order.js

```diff
@@ -278,10 +278,14 @@ function mutateRanksToAlphabetize(imported, order) {
 
   const groupRanks = Object.keys(groupedByRanks)
 
-  const sorterFn = order === 'asc' ? importsSorterAsc : importsSorterDesc
+  const sorterFn = alphabetizeOptions.order === 'asc' ? importsSorterAsc : importsSorterDesc
   // sort imports locally within their group
   groupRanks.forEach(function(groupRank) {
-    groupedByRanks[groupRank].sort(sorterFn)
+    groupedByRanks[groupRank].sort((a, b) => {
+      return sorterFn.apply(null, alphabetizeOptions.caseInsensitive
```

Comment:
rather than using apply here, i'm going to hoist the comparator out of the iteration